marklogic-data-hub-central/ui/src/components/column-selector/column-selector.test.tsx

```diff
@@ -27,19 +27,32 @@ describe('Column selector component', () => {
   });
 
   test('Verify cancel button closes popover', () => {
-    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns}/>);
+    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns} primaryKey={''}/>);
     const cancelButton = getByText('Cancel');
     cancelButton.onclick = jest.fn();
     fireEvent.click(cancelButton);
     expect(cancelButton.onclick).toHaveBeenCalledTimes(1);
   });
 
   test('Verify apply button closes popover', () => {
-    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns}/>);
+    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns} primaryKey={''}/>);
     const applyButton = getByText('Apply');
     applyButton.onclick = jest.fn();
     fireEvent.click(applyButton);
     expect(applyButton.onclick).toHaveBeenCalledTimes(1);
   });
+
+  test('Verify primaryKey tooltip renders', async () => {
```

Comment:
All these tests for popover and tooltips, we should group them together instead of adding 1 line tests.

Reply:
Merged tests.